ui/cypress/e2e/variables.test.ts

```diff
@@ -6,7 +6,7 @@ describe('Variables', () => {
 
     cy.signin().then(({body}) => {
       cy.wrap(body.org).as('org')
-      cy.visit(`organizations/${body.org.id}/variables_tab`)
+      cy.visit(`organizations/${body.org.id}/variables`)
```

Comment:
looks like youll need to update the other tests using org resource routes to remove the `_tab`